es-ds-components/components/es-data-table.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <DataTable
+        scrollable
+        :pt="{
+            table: {
+                class: 'font-size-75 font-size-md-100',
+                style: 'border-collapse: separate; border-spacing: 0;'
+            },
+            wrapper: { 
+                style: 'border-radius: inherit'
+            },
+            thead:{
+                style: 'top: 0px; z-index: 2'
+            }
+        }"
+        :value="props.items"
+        class="border table__data-table table-spacing table"
+        :scrollHeight="scrollHeight"
+    >
+    <Column v-for="(col,index) of columns" :key="col?.key ? col.key : col" :field="col?.key ? col.key : col" :header="label(col)" 
+        :sortable="col?.sortable"
+        :frozen="checkFrozenRow(index)" alignFrozen="left"
+        :pt="{
+            sortIcon:{
+                style: 'display: none'
+            }, 
+            headerCell: (options)  => ({
+                class:[
+                    {
+                        'stickyFirstCol': options.props.frozen,
```

Comment:
It seems like we also need to apply a class to make the cursor a pointer when hovering a sortable column

Reply:
In order to tell the user it's clickable and maintain the current behavior

Reply:
Good catch! I never noticed the pointers before!